spring-integration-gcp/src/main/java/org/springframework/integration/gcp/pubsub/outbound/PubSubMessageHandler.java

```diff
@@ -170,6 +177,19 @@ public void setPublishTimeout(long timeoutMillis) {
 		setPublishTimeoutExpression(new ValueExpression<>(timeoutMillis));
 	}
 
+	public ListenableFutureCallback<String> getPublishFutureCallback() {
```

Comment:
Why do you always add these `public` getters?
When I use `PubSubMessageHandler` from IDE declaring bean, I have an extra noise for unexpected getters.
I somehow think there is just enough to let end-user to see what (s)he expects - setters.
I don't mind to have getters for properties, but let's make them `protected` for unusual extensions.